src/postgres_client/postgres_client_account_index.rs

```diff
@@ -0,0 +1,377 @@
+use {
+    super::{
+        DbAccountInfo, ReadableAccountInfo, SimplePostgresClient,
+        DEFAULT_ACCOUNTS_INSERT_BATCH_SIZE,
+    },
+    crate::{
+        accountsdb_plugin_postgres::{
+            AccountsDbPluginPostgresConfig, AccountsDbPluginPostgresError,
+        },
+        inline_spl_token::{self, GenericTokenAccount},
+        inline_spl_token_2022,
+    },
+    log::*,
+    postgres::{Client, Statement},
+    solana_accountsdb_plugin_interface::accountsdb_plugin_interface::AccountsDbPluginError,
+    solana_measure::measure::Measure,
+    solana_metrics::*,
+    solana_sdk::pubkey::Pubkey,
+    tokio_postgres::types,
+};
+
+const TOKEN_INDEX_COLUMN_COUNT: usize = 3;
+/// Model the secondary index for token owner and mint
```

Comment:
nit: This comment currently isn't very helpful, maybe an explanation of:
1. How this secondary index works
2. How each of the fields is necessary for this index

Reply:
Updated